.travis.yml

```diff
@@ -58,10 +58,8 @@ script:
         path/to/luacheck/bin/luacheck spec/*.lua
         mv src2 src
     else
-        VERSION=`git describe --tags | sed 's/\(.*\)-.*/\1/' | sed 's/[^0-9-]*//g' | \
-            cut -c 2- | sed 's#-#\.#g' | sed 's/\.[^.]*$//'`
-        RELEASE=`git describe --tags | sed 's/\(.*\)-.*/\1/' | sed 's/[^0-9-]*//g' | \
-            cut -c 2- | sed 's#-#\.#g' | cut -d '.' -f4`
+        VERSION=`git describe --tags | sed 's/\(.*\)-.*/\1/' | cut -d '-' -f-1`
+        RELEASE=`git describe --tags | sed 's/\(.*\)-.*/\1/' | cut -d '-' -f2`
```

Comment:
Please, update variables and comment above likewise we discussed in the lulpeg PR before.

Reply:
Ok, done.